pyproject.toml

```diff
@@ -26,6 +26,7 @@ cryptography = ">=41.0.6,<42.1.0"
 nats-py = { version = "2.6.0", optional = true }
 boto3 =  { version = ">=1.28.16,<1.29.0", optional = true }
 kafka-python = { version = ">=2.0.0,<2.1.0", optional = true }
+amqpstorm = {version = "^2.10.7", optional = true, extras = ["amqpstorm"]}
```

Comment:
Is there any library that is async already?

For example https://github.com/mosquito/aio-pika

Reply:
Prefer async libraries if possible.

Reply:
https://github.com/truefoundry/async_service/blob/main/async_processor/nats_pub_sub.py